Formula/i/intltool.rb

```diff
@@ -4,7 +4,7 @@ class Intltool < Formula
   url "https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz"
   sha256 "67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd"
   license "GPL-2.0-or-later"
-  revision 1
+  revision 2
```

Comment:
Does this need `revision i.e. do things break unless it is?

Reply:
yeah, it is already broken for linux build

Reply:
Already broken i.e. the Linux build is broken before this or without it? Ideally we'd only `revision` if every platform (including users building from source) have to reinstall for a critical change.

Reply:
it is broken after the perl upgrade.